apps/vessel-history/features/resources/resources.slice.ts

```diff
@@ -25,6 +25,13 @@ export const selectResources = createSelector([originalSelectResource], (resourc
   return Object.keys(resources)
     .map((url) => {
       const resource = resources[url]
+      // We remove gaps where there is non intentional disabling
+      const eventsWithGapsNonIntentionalDisablingRemoved =
+        Array.isArray(resource.data) &&
+        (resource.data as any[])
+          .filter((event) => event.type !== EventTypes.Gap || event.gap.intentionalDisabling === true)
+
+
```

Comment:
WDYT of defining a filter
```suggestion
      const excludeNonIntentionalDisablingGaps = (event) => event.type !== EventTypes.Gap || event.gap.intentionalDisabling === true

```